code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -207,6 +207,7 @@
 	emote_see = list("honks", "sweats", "jiggles", "contemplates its existence")
 	speak_chance = 5
 	dextrous = TRUE
+	hud_type = /datum/hud/dextrous
```

Comment:
Likely on accident, but you're giving the dexterous hud to fleshclowns. 🤡 🍖

Reply:
They are apparently supposed to have it, from the ``dextrous`` var above being true

Reply:
Oh, you're right. I totally glossed over it.